xdsl/transforms/experimental/convert_stencil_to_ll_mlir.py

```diff
@@ -340,7 +316,7 @@ def match_and_rewrite(self, op: BufferOp, rewriter: PatternRewriter, /):
 
 @dataclass
 class ApplyOpToParallel(RewritePattern):
-    return_targets: dict[ReturnOp, list[SSAValue | None]]
+    return_targets: dict[ApplyOp, list[SSAValue | None]]
```

Comment:
why this change?

Reply:
It just made things simpler in one of the pattern :)
They are sort of interchangeable, as one is the only terminator of the other, they have to exist in unique pairs!